lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'array.pop.pop'
 
```

Comment:
Close, but this isn't doing what it looks like it's doing.
When we call pop on array, the value returned is the last element (a FixNum)
when we try to call the second pop, it's not being called on the array, but rather what was last returned
this will throw an error because Fixnum has no pop method!
You could do 
`array.pop
array.pop
`
or more idiomatic, `array.pop 2`